docs/docs/working-with-images.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Working With Images In Gatsby
+---
+
+# Working With Images In Gatsby
+
+Gatsby has several useful [plugins](/docs/plugins/) that work together for adding and optimizing images for [page components](/docs/building-with-components/#page-components)
+
+
+The recommend approach is to 
+* GraphQL to query image files
+* Sharp, an image processing library, to transform images
+* The Gat
+
+
+## Access Your Images With GraphQL
+
+[GraphQL can access image files](docs/querying-with-graphql/#images) with the [`gatsby-source-filesystem`](/packages/gatsby-source-filesystem/) plugin. 
+
+Once you have this installed you should be able to query the files as shown in the our tutorial on [writing queries](tutorial/part-four/#build-a-page-with-a-graphql-query). 
```

Comment:
*the tutorial

Also, the file path might have changed for that part of the tutorial! We just split part 4 into four smaller tutorials (same content, just divided up into digestible chunks (hopefully)).